examples/bzlmod/MODULE.bazel

```diff
@@ -158,6 +158,23 @@ pip.whl_mods(
 )
 use_repo(pip, "whl_mods_hub")
 
+# Because below we are using `windows_aarch64` platform, we have to define various
+# properties for it.
+pip.default(
+    arch_name = "aarch64",
+    config_settings = [
+        "@platforms//os:windows",
+        "@platforms//cpu:aarch64",
+    ],
+    env = {
+        "platform_version": "0",
+    },
+    os_name = "windows",
+    platform = "windows_aarch64",
+    platform_tags = ["win_amd64"],
+    want_abis = [],  # default to all ABIs
```

Comment:
I don't like the name "want_abis". "want" is a weak/ambiguous word that implies its expected/common it won't be fulfilled -- does it mean "requires" or "if available" ?

allowed_abis, requires_abi, restricted_to_abis -- better communicates it must be one of the values

preferred_abis -- better communicates none may be chosen.

Reply:
I'll go for `whl_abi_tags` and `whl_platform_tags`.